azure-kusto-data/azure/kusto/data/security.py

```diff
@@ -10,6 +10,7 @@
 from adal import AuthenticationContext, AdalError
 from adal.constants import TokenResponseFields, OAuth2DeviceCodeResponseParameters, OAuth2ResponseParameters
 from azure.identity import ManagedIdentityCredential
+from azure.core.credentials import AccessToken
```

Comment:
@ArielYehezkely ,
Question: azure.core.credentials is a dependency of azure.identity which I use only to specific the return value of a function.
Can I avoid specifying it in setup.py 'install_requires'?